python/ribasim/ribasim/db_utils.py

```diff
@@ -20,10 +20,12 @@ def exists(connection: Connection, name: str) -> bool:
 
 
 def _set_gpkg_attribute_table(connection: Connection, table: str) -> None:
+    from ribasim import fake_date
```

Comment:
Is is really preferred style to put these internal imports inside functions? Or is there another reason to do it like this?